py/web/views/events.py

```diff
@@ -901,22 +901,32 @@ async def execute(self, m: Model):
         )
 
 
+class GroupUpdateMessage(BaseModel):
+    ticketType: int
+    message: str
+
+
 class EventUpdate(UpdateView):
     class Model(BaseModel):
         subject: constr(max_length=200)
-        message: str
+        message: Optional[str]
+        groupMessages: Optional[List[GroupUpdateMessage]]
```

Comment:
Try to avoid camel case in variable names in python
```suggestion
        group_messages: List[GroupUpdateMessage] = Field(None, alias='groupMessages')
```